modules/custom/az_demo/data/az_demo_node_publication.json

```
@@ -0,0 +1,284 @@
+{
+  "publications": [
+    {
+      "id": 1,
+      "title": "The Most Fearsome Life of the Great Gargantua, Father of Pantagruel.",
+      "authors": "10",
+      "accessed_date": "2023-07-20",
+      "abstract": "A detailed account of the life and adventures of Gargantua.",
+      "approximate": "false",
+      "category": "Biography",
+      "container": "Book",
+      "date": "2022-06-01",
+      "date_type": "default",
+      "doi": "10.1000/182",
+      "extra_links": "https://example.com/gargantua",
+      "image": "gargantua.jpg",
```

I'm not sure if the image files referenced in this JSON file were meant to be included with this PR, but I'm not seeing any images attached to the demo publications. If these image files won't be included, then it might be best to leave the `image` property blank for these publications.

```suggestion
      "image": "",
```